lib/MatterbridgeManager.php

```diff
@@ -666,28 +665,51 @@ private function launchMatterbridge(Room $room): int {
 
 	/**
 	 * kill the mattermost processes (owned by web server unix user) that do not match with any room
+	 * @param bool $killAll
 	 */
-	public function killZombieBridges(): void {
+	public function killZombieBridges(bool $killAll = false): void {
```

Comment:
This could be named `killBridges` and have a `$zombieOnly` boolean parameter. This would make it less confusive when it's called from `stopAllBridges`.